src/Exception/RuntimeException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Drupal\os2web_key\Exception;
+
+/**
+ * Runtime exception.
+ */
+class RuntimeException extends \RuntimeException {
```

Comment:
Would it be better to rename this exception to something like `ModuleRuntimeExeception` or `SSLRuntimeException`?

Reply:
That would not add any value. The `RuntimeException` is namespaced and can be aliased (`use … as …`) if need be.